spring-kafka/src/main/java/org/springframework/kafka/core/ConsumerFactory.java

```diff
@@ -63,16 +63,13 @@
 	 * {@code client.id} property, if present.
 	 * If a factory does not implement this method, {@link #createConsumer(String, String)}
 	 * is invoked, ignoring the prefix.
-	 * TODO: remove default in 2.2
```

Comment:
Hehe - this TODO conflicts with https://github.com/spring-projects/spring-kafka/issues/598